scripts/install-cni.sh

```diff
@@ -14,6 +14,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+calico_ready() {
+  echo "Listing items matching /host/etc/cni/net.d/*calico*.conflist"
+  compgen -G "/host/etc/cni/net.d/*calico*.conflist"
```

Comment:
would be great to add a quick explanation that some errors during bootstrap are expected.